packages/msgpack/msgpack.1.2.1/descr

```diff
@@ -0,0 +1,4 @@
+Msgpack library for Objective Caml
```

Comment:
Can you rename that to `OCaml`? That's the official since a few years now :-) Thanks!